.github/workflows/self-scheduled.yml

```diff
@@ -15,6 +15,19 @@ on:
       slack_report_channel:
         required: true
         type: string
+      runner:
+        required: true
+        type: string
+      docker:
+        required: true
+        type: string
+      ci_event:
+        required: true
+        type: string
```

Comment:
to show things like

- Results of the Daily CI tests.
- Results of the Nightly CI tests.
- Results of the Past CI tests.